src/ignore_path.rs

```diff
@@ -11,7 +11,7 @@ impl IgnorePathSet {
         let mut ignore_builder = gitignore::GitignoreBuilder::new(ignore_list.rustfmt_toml_path());
 
         for ignore_path in ignore_list {
-            ignore_builder.add_line(None, ignore_path.to_str().unwrap())?;
+            ignore_builder.add_line(None, &ignore_path.to_str().unwrap().replace("\\", "/"))?;
```

Comment:
Could you fix this line so that `replace("\\", "/")` is called only on Windows? This could break in Unix & Linux where a file name can contain`\`:

```shell
mkdir /tmp/\\crazy
```

Reply:
Good point, thanks @topecongiro!